.travis.yml

```diff
@@ -10,7 +10,7 @@ before_script:
     - "sudo chmod 4755 /opt/google/chrome/chrome-sandbox"
 before_install:
   - npm cache clean --force
-script: npm run lint && npm run test
+script: npm run test:angular-versions
```

Comment:
This all looks great! Do you mind putting the lint verification back in place? It is nice to have for the PR check.

Reply:
it's here, but deep inside. for every version it executes `npm run build:all` which is `npm run lint && npm run test && npm run build`

Reply:
and thanks for the invitation!

Reply:
Thanks for all of your help!